### PR TITLE
 Make none_rec_action respect timid. #3242 

### PR DESCRIPTION
**Pull request by RollingStar**

### CODE REVIEW

File: beets/ui/commands.py [hunk 1]
```diff
@@ -496,6 +496,9 @@ def _summary_judgment(rec):
             'asis': importer.action.ASIS,
             'ask': None,
         })
+        # prompt the user if timid is enabled
+        if config['import']['timid'] and action == importer.action.ASIS:
```
Comment (RollingStar) [inline beets/ui/commands.py]: Is `importer.action.ASIS` the proper tool to use here? The as_choice function from beetbox/confuse complicates things.

Comment (sampsyo) [reply beets/ui/commands.py]: Yep, looks right to me.

**Comment by sampsyo** [comment]
Thanks for getting this started! I would have expected a somewhat stronger "override," though—i.e., the importer would ask you every time in timid mode, even if the recommendation is to "skip." That also has a somewhat simpler implementation—the first `if` in the `elif` cascade here would just say:

```py
if config['import']['timid']:
    return None
```

…and then proceed on to the other cases. Would that be too aggressive for your use case?

**Comment by RollingStar** [comment]
I viewed timid as "prompt before changing anything", not "prompt before doing anything". [The documentation](https://beets.readthedocs.io/en/latest/reference/config.html#timid) and [commands](https://github.com/beetbox/beets/blob/96d61b8b9305957b1f4c438386ac97fc6a5c7571/beets/ui/commands.py#L1011) lean towards your approach:

>Either yes or no, controlling whether the importer runs in timid mode, in which it asks for confirmation on every autotagging match, even the ones that seem very close. Defaults to no. The -t command-line flag controls the same setting.

>help=u'always confirm all actions'

**Comment by RollingStar** [comment]
Addressed your feedback with a new commit.

**Comment by sampsyo** [comment]
Looks perfect; thank you!! :sparkles:

**Pull request merged**